Diverse/Fuzzer.cs

```diff
@@ -60,6 +60,8 @@ public Fuzzer(int? seed = null, string name = null)
 
             _internalRandom = new Random(seed.Value);
 
+            _fuzzStrings = new FuzzerStrings(_internalRandom);
```

Comment:
FYI I will instead provide a reference to this as a IProvideCorePrimitivesToFuzzer to the FuzzerStrings (BTW, I would like to rename it StringFuzzer if you don't mind)